docs/guides/advanced/caching.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Caching
+---
+
+# Caching formula and fetcher results
+
+For performance reasons Coda uses a variety of caching techniques to store previous results and reuse them later. 
+
+
+## Formula cache {: #formula}
+
+When Coda's formula engine decides your [Pack formula][formulas] needs to be [recalculated][formulas_recalculation], it first checks a cache of previous formula results. If a matching entry is found, it is returned instead of re-running your code. When a cached result is returned, the Pack Maker Tools logs entry will end with **returned a prior result from the cache** and the **Cache hit** field will be set to **true**.
```

Comment:
Up to you if it's worth explaining, but technically it checks the cache on every formula invocation, not just recalcs/invalidations. If you add the formula FOO(2) and then elsewhere add the formula FOO(1 + 1) the second one will get a cached result even though it's a new/different formula, being invoked for the first time.

Reply:
Simplified the language a bit to avoid the longer discussion, and move the recalculation pointer earlier.